voucher/rest.py

```diff
@@ -121,6 +162,43 @@ def use_vouchers(self, request, username=None):
         return Response([UseLogSerializer(p).data for p in pending_transactions], status=status.HTTP_201_CREATED)
 
 
+class RegisterLogViewSet(viewsets.ModelViewSet):
```

Comment:
Ja, jeg vet. Poenget mitt var bare at `CoffeeLog` ikke sier noe om hva som egentlig logges, annet enn at det er relevant for `Coffee`